src/core/thread/meshcop_dataset_manager.cpp

```diff
@@ -387,6 +388,7 @@ ThreadError DatasetManager::Set(Coap::Header &aHeader, Message &aMessage, const
     }
 
     mNetwork = mLocal;
+    mNetwork.Store();
```

Comment:
`mNetwork` and `mLocal` are the same here, so I think this ok.  But in general, I think we should be storing the `mLocal` copy.